Firebase/Messaging/FIRMessagingContextManagerService.m

```diff
@@ -130,6 +130,7 @@ + (BOOL)handleContextManagerLocalTimeMessage:(NSDictionary *)message {
 
 + (void)scheduleLocalNotificationForMessage:(NSDictionary *)message
                                      atDate:(NSDate *)date {
+#if TARGET_OS_IOS
```

Comment:
If this method does nothing on tvOS, it should be marked `__TVOS_UNAVAILABLE` in the header file.

Reply:
Even if the header is marked as unavailable, we will need to mark the implementation with this macro.

This is a private implementation and there are quite some private methods which are not available for tvOS. I'm not sure if we do the same for other community support at this point. This CL is just to ensure the code compiles under tvOS. We might start mark __TVOS_UNAVAILABLE in the public API first.